modules/ROOT/pages/_partials/rn-known-issues-ref.adoc

```diff
@@ -3,15 +3,36 @@
 
 This section lists known issues with  {PRODUCT} {MINOR_VERSION_NO}:
 
```

Comment:
Just a reminder to add a "known issue" not with workaround for https://issues.redhat.com/browse/INTLY-9312

Reply:
Done.